docs/getting-started/installing-substrate.md

```diff
@@ -2,37 +2,44 @@
 title: "Installing Substrate"
 ---
 
+Following ways to install Substrate assume you are running Mac OS, Arch, or a Debian-based OS like Ubuntu. For Windows users, you can try it [here](https://github.com/paritytech/substrate#612-windows).
+
+## Fast Installation
+
+If you plan to create your own blockchain, then run this simple one-liner:
+
+```bash
+curl https://getsubstrate.io -sSf | bash -s -- --fast
+```
+
+This command will install the [substrate-up](https://github.com/paritytech/substrate-up) commands for creating your own nodes and modules.
+
+This is the most common way to start your Substrate's journey.
+
 ## Full Installation
 
-If you're running Mac OS, Arch, or a Debian-based OS like Ubuntu, then **with a good internet connection** run this simple one-liner:
+If you want to experience the latest Substrate's features and **have a good internet connection**, then run command:
```

Comment:
So this should simply say that if you want to have a local copy of the latest Substrate Testnet Node and a copy of Subkey, you should install Substrate without the `--fast` flag.

Reply:
The `Substrate Testnet` confused me a bit, if we view Substrate as a framework, there is less value to emphasis a testnet. 
I have done a refactor with the suggestion "without the `--fast flag`".

Reply:
What I mean by "substrate testnet node" is the contents in this folder: https://github.com/paritytech/substrate/tree/master/node

This folder is a working node, which if you run, will connect you to Flaming Fir, the Substrate Testnet.

When you install without the fast flag, we produce a binary called "substrate" which if you run, will connect you to that test net.

Reply:
Thanks for the clear explanation. 
The value of substrate command which based the node is to help user try the Substrate framework, no more other values IMO. Do we really want to detail the Substrate Testnet in the `Full Installation` section?
I also don't think testnet is a good word for such thing, I prefer testbed or blockchain demo.